tests/conftest.py

```diff
@@ -30,25 +48,25 @@ def fixture_unknown_user(variables, org):
     return User(**variables[org]['users']['unknown'])
 
 
-@pytest.fixture(name='user')
+@pytest.fixture(name='user', scope='session')
 def fixture_user(variables, org):
     """Return a registered user."""
     return User(**variables[org]['users']['ashley'])
 
 
-@pytest.fixture(name='server_url')
-def fixture_server_url(request):
-    """Return the URL to the Redash server."""
-    return request.config.option.server_url
-
-
 @pytest.fixture(name='login_page')
 def fixture_login_page(selenium, server_url, org):
     """Return a page object model for the login page."""
     login_page = LoginPage(selenium, server_url, org=org)
     return login_page.open()
 
 
+@pytest.fixture(name='server_url')
```

Comment:
I think it helps comprehensibility if we define pytest fixtures in the order in which they are being requested. I suggest to revert this particular change to ``fixture_server_url``.

Reply:
So I should just move it back to line 39?

Reply:
Yes, please! 👍